lib/elastic_record/index/deferred.rb

```diff
@@ -57,15 +59,19 @@ def flush_deferred_actions!
             deferred_actions.clear
           end
 
-          def search_request(method, args)
+          def index_from_request(method, args)
             if method == :json_get && args.first =~ /^\/(.*)\/_m?search/
               $1.partition('/').first
             end
```

Comment:
`search_after` requests that use a PIT don't include the index in the URI, so we won't know which index to refresh.

Reply:
The only place I see a URL that matches this is https://github.com/data-axle/elastic_record/blob/7ff5eac599bbbf49a6df8c658927c98d33ba9ca1/lib/elastic_record/index/pagination.rb#L19.  Where do we POST `_pit` without the index in the URI?

Reply:
You're right, I was thinking this refresh happens on every call to `_search` using the PIT. 

I like your approach, there probably shouldn't need to be refreshes on every `search_after` call anyway, since we don't need to test Elasticsearch's PIT functionality.